docs/code/IdvFileFormat.md

```diff
@@ -0,0 +1,191 @@
+# IDV File Format
+
+This document describes ML.NET's Binary dataview file format, version 1.1.1.5
```

Comment:
This looks familiar... was this intended to be part of this PR? Was there a bad merge somewhere?